protocol/messenger_handler.go

```diff
@@ -292,7 +292,7 @@ func (m *Messenger) createMessageNotification(chat *Chat, messageState *Received
 	notification := &ActivityCenterNotification{
 		ID:          types.FromHex(chat.ID),
 		Name:        chat.Name,
-		LastMessage: message,
+		Message:     message,
```

Comment:
Here we're changing the creation of the `NewOneToOne` and `NewPrivateGroupChat` notifications to use the `Message` field instead of the `LastMessage` field.